README.md

```
@@ -1,390 +1,394 @@
-)
+## Error handling
+
+Every error descends from `ZammadAPI::Error`.
 
-ticket.id # id of record
-ticket.number # uniq number of ticket
-ticket.title # 'a new ticket #1'
-ticket.group # 'Support'
-ticket.created_at # '2022-01-01T12:42:01Z'
-# ...
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)